server/routes/api.js

```diff
@@ -14,7 +14,7 @@ router.get('/articles', function (req, res, next) {
             res.json(articles)
         }
     };
-    articleClient.findArticles({ visible: true }, callBack)
+    articleClient.findArticles({ visible: true,language:"En" }, callBack)
```

Comment:
could you instead try and get language from `req.params` and if nothing is provided then default to 'En'

Reply:
How can I get language from req.params in API?

Reply:
in the url, you can send `/api/articles?language=En` - you can read it as `req.params.language`